manage.py

```diff
@@ -56,11 +54,27 @@ def add_fake_data(number_users):
     Adds fake data to the database.
     """
     User.generate_fake(count=number_users)
-    RoomRequest.generate_fake()
+    RoomRequest.generate_fake()    
+
+@manager.option(
+    '-n',
+    '--id',
+    type=int,
+    help='id of the user you want to delete',
+    dest='id')
+    # something with routing here??
+    # @app.route(/room-request/<id>/delete)
+    # ^ probably replace app with something else?
```

Comment:
You don't need any routing here - this is if we ran:
```
python manage.py delete <id>
```
I guess this is fine for testing purposes! Did you try this?